openpype/hosts/maya/plugins/create/convert_legacy.py

```diff
@@ -135,6 +139,18 @@ def _convert_per_renderlayer(self, instance_nodes, data, creator):
             # "rendering" family being converted to "renderlayer" family)
             original_data["family"] = creator.family
 
+            # recreate subset name as without it would be
+            # `renderingMain` vs correct `renderMain`
+            project_name = get_current_project_name()
+            asset_doc = get_asset_by_name(project_name,
+                                          original_data["asset"])
+            subset_name = get_subset_name("render",
+                                          original_data["variant"],
+                                          data["task"],
+                                          asset_doc,
+                                          project_name)
```

Comment:
```suggestion
            subset_name = creator.get_subset_name(
                original_data["variant"],
                data["task"],
                asset_doc,
                project_name)
```

Reply:
This won't work as `creator.family` != `render` because I don't know why.

Reply:
Now it does

Reply:
This suggestion needs to still be committed.